llvm/test/CodeGen/NVPTX/compute-ptx-value-vts.ll

```diff
@@ -0,0 +1,45 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_20
+
+define <6 x half> @half6() {
+  ret <6 x half> zeroinitializer
+}
+
+define <10 x half> @half10() {
+  ret <10 x half> zeroinitializer
+}
+
+define <14 x half> @half14() {
+  ret <14 x half> zeroinitializer
+}
+
+define <18 x half> @half18() {
+  ret <18 x half> zeroinitializer
+}
+
+define <998 x half> @half998() {
```

Comment:
Is there any particular significance about the 998 elements?

I think the interesting cases are the few powers of two, and adjacent values above and below it.
It does not make much sense to go above `<9 x half>`, as only the vectors up to 128bit in size have special treatment.

Reply:
There isn't any particular significance to `998`. I just wanted to add a test case to prevent any special treatment that could occur. However, I didn't know that, "only the vectors up to 128bit in size have special treatment." In light of this, I've removed the tests.